tutordiscovery/plugin.py

```diff
@@ -25,6 +25,7 @@
         "OAUTH2_KEY_DEV": "discovery-dev",
         "OAUTH2_KEY_SSO": "discovery-sso",
         "OAUTH2_KEY_SSO_DEV": "discovery-sso-dev",
+        "CACHE_REDIS_DB": 1,
```

Comment:
I initially thought that we should use a different DB than for the openedx cache, but since we use a different prefix it makes sense to use the same one. I'd just like to suggest the following, to be sure that we default to the same DB:

`"CACHE_REDIS_DB": "{{ OPENEDX_CACHE_REDIS_DB }}",`

Thus, the new plugin will have to be released after tutor is upgraded.

Reply:
Yes, a new release is needed if depends on "OPENEDX_CACHE_REDIS_DB". I will update the PR.